salt/states/boto_lc.py

```diff
@@ -185,9 +186,10 @@ def present(
             Indicates what volume type to use. Valid values are standard, io1, gp2.
             Default is standard.
 
-        delete_on_termination
-            Indicates whether to delete the volume on instance termination (true) or
-            not (false).
+    delete_on_termination
+        Indicates whether to delete the volume on instance termination (true) or
+        not (false). Default is "None" which corresponds to not specified.
+        Amazon has different defaults for root and additional volumes.
```

Comment:
To be pedantic for the sake of least surprise for end users, I'd recommend something like
```
Whether the volume should be explicitly marked for deletion when its instance is
terminated (True), or left around (False).  If not provided, or None is explicitly passed,
the default AWS behaviour is used, which is True for ROOT volumes of instances, and
False for all others.
```

Or something to that effect... :)

Reply:
Sounds good, changed in last commit.